cookbook/packages/package_json_example.json

```diff
@@ -0,0 +1,28 @@
+{
+    "proposal": {

```

Comment:
call this package default and add it at the init_chain_commands.sh

Reply:
done